extension/public/worker/facebook.js

```diff
@@ -0,0 +1,33 @@
+/** @param {HTMLImageElement} img */
+function blurImage(img) {
+    img.style.filter = "blur(30px)";
+}
+
+
+/** @param {HTMLDivElement} el */
+function checkPost(el) {
+    if (window.matchesBlocklist(el.innerText)) {
+      el.style.filter = "blur(4px)";
+    }
+}
+  
+/** @param {HTMLImageElement} img */
+function checkImage(img) {
+    if (img.alt && window.matchesBlocklist(img.alt)) {
+      blurImage(img);
+    }
+}
+
+window.onceReady = () => {
+// quit if facebook is not enabled
+    if (!window.enabled.facebook) return;
+  
+    // check every 1 second if a new post has been loaded
+    setInterval(() => {
+      // check every visible facebook post, which can be identified since it's a
+      // div with an attribute called lang="en". Politicry only supports English (for now)
+      document.querySelectorAll('div[lang="en"]').forEach(checkPost);
```

Comment:
```suggestion
      document.querySelectorAll('div[data-ad-preview="message"]').forEach(checkPost);
```

`div[lang="en"]` doesn't seem to work for facebook, a different selector like `[data-ad-preview="message"]` might work

All the other classes and attributes in facebook seem to be automatically generated

![image](https://user-images.githubusercontent.com/16009897/194482270-9206470b-71ff-45cf-92ee-459c1b15ca75.png)